var/spack/repos/builtin/packages/gpi-2/package.py

```diff
@@ -60,7 +60,8 @@ class Gpi2(AutotoolsPackage):
     depends_on('libtool',  type='build', when='@1.4.0:')
     depends_on('m4',       type='build', when='@1.4.0:')
 
-    depends_on('gawk', type='run')
+    depends_on('sed', type=('build','run'))
+    depends_on('gawk', type=('build','run'))
```

Comment:
Your description states these are `build` dependencies but you are adding `run` here.  Is it one or both?

Reply:
@tldahlgren, you are right, the above message, and the commit, was not completely precise. I have corrected both of them to give account of the actual situation.